v1.0/multi-active-availability.md

```diff
@@ -62,6 +62,6 @@ For this example, we have 3 CockroachDB nodes (**A**, **B**, **C**) in a multi-a
 
 ## What's next?
 
-To get a greater understanding of how CockroachDB is a survivable system that enforces strong consistency, check out our [architecture documentation](/architecture/overview.html).
+To get a greater understanding of how CockroachDB is a survivable system that enforces strong consistency, check out our [architecture documentation]({{ '/architecture/overview.html' | relative_url }}).
```

Comment:
Thanks for catching this one `htmltest` and Nikhil! So whenever we link to a page in the same version but in another directory, we need to do this?

Reply:
Yep! I'm surprised HTML proofer didn't catch this, actually.